Source/Core/Core/PowerPC/BreakPoints.h

```diff
@@ -69,8 +77,26 @@ class BreakPoints
   void Clear();
   void ClearAllTemporary();
 
+  void SetBreakpointTriggered(bool b, u64 addr = -1, MemCheckCondition cond = MEMCHECK_NONE)
+  {
+      m_breakpointTriggered = b;
+      m_breakpointAddress = addr;
+      m_breakpointCondition = cond;
+  };
+  bool GetBreakpointTriggered() { return m_breakpointTriggered; };
+  bool GetBreakpointTriggered(u64& addr, MemCheckCondition& cond)
+  {
+      addr = m_breakpointAddress;
+      cond = m_breakpointCondition;
+      return m_breakpointTriggered;
+  };
+
 private:
   TBreakPoints m_breakpoints;
+
+  bool m_breakpointTriggered;
```

Comment:
Our naming convention for member variables is snake case, so this should be `m_breakpoint_triggered`. 

Please read our [contributing guide](https://github.com/dolphin-emu/dolphin/tree/master/Contributing.md) for more information.